test/integration/generated/additionalProperties/src/models/mappers.ts

```diff
@@ -9,7 +9,6 @@
 import * as coreHttp from "@azure/core-http";
 
 export const PetAPTrue: coreHttp.CompositeMapper = {
-  serializedName: "PetAPTrue",
```

Comment:
This is an intended change. We have been falling back to the schema name when serialized name is not defined. However falling back to the name doesn't add value and isn't needed(except for xml scenarios), it is only required for properties for properly mapping and sending data over the wire. Hence I'm removing unnecessary serializedName properties from mappers